### ISSUE TITLE
potential grid bug with `.remove()`

### ISSUE THREAD

**Issue by tyleretters** [monome/norns#1220]
```
require("tabutil")

g = grid.connect()

print("================ before")
tabutil.print(g)

function g.remove()
  print("shouldn't this print when i unplug the grid?")  
end

print("================ after")
tabutil.print(g)
```

matron output is:

```
================ before
all function: 0x3af5a0
name  monome 128 m1001146
rows  8
remove  function: 0x549cd0
led function: 0x3af698
intensity function: 0x3af220
cols  16
rotation  function: 0x3af480
device  table: 0x4cbc50
refresh function: 0x3af438
================ after
all function: 0x3af5a0
name  monome 128 m1001146
rows  8
remove  function: 0x3592f8
led function: 0x3af698
intensity function: 0x3af220
cols  16
rotation  function: 0x3af480
device  table: 0x4cbc50
refresh function: 0x3af438
```

we can see the remove function is indeed updated, but `g.remove()` never fires.

**Comment by catfact** [comment]
looks like the `Grid.remove()` method was not updated when vports were added.

**Comment by tehn** [comment]
fixed #1226
